src/main/resources/db/migration/V208__add_new_column_epim_id.sql

```diff
@@ -0,0 +1,18 @@
+-- FACT-1903
+-- Add new column epim_id to court address table
+-- Add check constraint to ensure epim_id length is less than or equal to 30
+ALTER TABLE search_courtaddress ADD COLUMN IF NOT EXISTS epim_id VARCHAR(30);
+DO $$
+BEGIN
```

Comment:
Could you explain how this works? Looks very GPT generated

Reply:
this but just adds a constraint if there isn't one there already by checking the info schema. i don't think you can use 'if not exists' directly in the alter table query for some reason and i tend to like using if not exists because if somehow it was ran multiple times it won't error, it just skips over it